modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/util/MBMessageFormatUploadHandlerProviderRegistryUtil.java

```diff
@@ -3,38 +3,36 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-package com.liferay.message.boards.web.internal.upload.format;
+package com.liferay.message.boards.web.internal.util;
 
+import com.liferay.message.boards.web.internal.upload.format.MBMessageFormatUploadHandler;
 import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerMap;
 import com.liferay.osgi.service.tracker.collections.map.ServiceTrackerMapFactory;
 
+import org.osgi.framework.Bundle;
 import org.osgi.framework.BundleContext;
-import org.osgi.service.component.annotations.Activate;
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.framework.FrameworkUtil;
 
 /**
  * @author Alejandro Tardín
  */
-@Component(service = MBMessageFormatUploadHandlerProvider.class)
-public class MBMessageFormatUploadHandlerProvider {
+public class MBMessageFormatUploadHandlerProviderRegistryUtil {
```

Comment:
Hey @GabrielHenriquedev, rename it to `MBMessageFormatUploadHandlerRegistryUtil`.